subprojects/docs/src/docs/userguide/buildCache.adoc

```diff
@@ -32,6 +32,8 @@ When using a shared build cache for task output caching this even works across d
 
 Apart from task output caching, we expect other features to use the build cache in the future.
 
+NOTE: A complete guide is available about https://guides.gradle.org/using-build-cache/[configuring and using the build cache]. It covers the different scenarios caching can improve, and detailed discussions of the different caveats you need to be aware of when enabling caching for a build.
```

Comment:
"Configuring" could be the configuration of normalization in your build. Setting up the service is also part of configuring the caching.